src/Analyser/MutatingScope.php

```diff
@@ -4186,9 +4186,9 @@ private function intersectConditionalExpressions(array $otherConditionalExpressi
 
 	/**
 	 * @param array<string, ConditionalExpressionHolder[]> $conditionalExpressions
-	 * @param array<string, VariableTypeHolder> $variableTypes
-	 * @param array<string, VariableTypeHolder> $theirVariableTypes
-	 * @param array<string, VariableTypeHolder> $mergedVariableHolders
+	 * @param array<int|string, VariableTypeHolder> $variableTypes
+	 * @param array<int|string, VariableTypeHolder> $theirVariableTypes
+	 * @param array<int|string, VariableTypeHolder> $mergedVariableHolders
```

Comment:
Very sad, but key of exprString is sometimes integer, because of PHP's bad behavior...
https://3v4l.org/5UOnT